apps/dapp/src/components/GlobalFooter.tsx

```diff
@@ -0,0 +1,128 @@
+  label: string;
+  href: string;
+  isExternal?: boolean;
+}
+
+const FooterLinks1: FooterLink[] = [
+  {
+    label: 'Terms and Conditions',
+    href: '/page/terms-and-conditions',
+  },
+  {
+    label: 'Privacy Policy',
+    href: '/page/privacy-policy',
+  },
+  // {
+  //   label: "Technology FAQs",
+  //   href: "/",
+  // },
+];
+const FooterLinks2: FooterLink[] = [
+  {
+    label: 'Discord',
+    href: 'https://discord.com/invite/Wx8yAu3JKS',
+    isExternal: true,
+  },
+  {
+    label: 'X (Formerly Twitter)',
+    href: 'https://twitter.com/itsdropaudio',
+    isExternal: true,
+  },
+  {
+    label: 'Instagram',
+    href: 'https://www.instagram.com/itsdropaudio',
+    isExternal: true,
+  },
+  {
+    label: 'News/Blog',
+    href: '/blog',
+  },
+];
+
+const FooterLinks3: FooterLink[] = [
+  {
+    label: 'Learn',
+    href: '/blog/welcome-to-drop',
+  },
+  {
+    label: 'Apply',
+    href: 'https://dropaudio.typeform.com/jointhequeue',
+    isExternal: true,
+  },
+];
+
+const NavList = ({ links, title }: { links: FooterLink[]; title?: string }) => {
+  return (
+    <nav>
+      {title && <h1 className="font-bold">{title}</h1>}
+      {links.map((link, index) => (
+        <div key={link.label + index}>
+          {link.isExternal ? (
+            <a
+              href={link.href}
+              target="_blank"
+              rel="noreferrer"
+              className="opacity-60 hover:opacity-100 text-sm"
+            >
+              {link.label}
+            </a>
+          ) : (
+            <Link
+              href={link.href}
+              className="opacity-60 hover:opacity-100 text-sm"
+            >
+              {link.label}
+            </Link>
+          )}
+        </div>
+      ))}
+    </nav>
+  );
+};
+
+export const GlobalFooter = () => {
+  return (
+    <div className="bg-black text-white relative px-8 pt-20 overflow-hidden">
+      <div className="space-y-12 md:space-y-6">
+        <div className="flex items-center justify-center">
+          <SocialLinks />
+        </div>
+
+        <div className="container p-6 mx-auto">
+          <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
+            <div className="flex justify-center text-center md:text-left">
+              <NavList links={FooterLinks1} title="Boring Stuff" />
+            </div>
+            <div className="flex justify-center text-center md:text-left">
+              <NavList links={FooterLinks2} title="Community" />
+            </div>
+            <div className="flex justify-center text-center md:text-left">
+              <NavList links={FooterLinks3} title="Artist" />
+            </div>
+          </div>
+        </div>
+
+        <div className="max-w-xl mx-auto">
+          <SubscribeForm />
+        </div>
+
+        <div className="flex items-center justify-center pt-10">
+          <Image
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+            src={DropLogoWhite}

```

Comment:
NIT: Let's remove the "Drop" language so that it is easy for someone to come in and add their own brand. Maybe just call this `LogoWhite`